### DISCUSSION TITLE
Why a class inherits an interface does not need implement the interface' methods?

### DISCUSSION THREAD

**Discussion by giantpenguin** [OrchardCMS/OrchardCore#14398] [Q&A] [answered]
Why a class inherits an interface does not need implement the interface' methods? How has been Orchard implemented to do this? 

Looking into the Lombiq TrainningDemo the DisplayManagementController  : Controller, IUpdateModel but nowhere in the DisplayManagementController implement the IUpdateModel methods.

**Comment by jtkech** [comment, marked as answer]
Because it inherits from `Controller` that inherits from `ControllerBase` that implements `IUpdateModel` methods.

**Reply by giantpenguin** [reply]
Thank @jtkech very much! 
You helped me to realize that the OrchardCore.DisplayManagement.ModelBinding.IUpdateModel defines its methods which are from the Microsoft.AspNetCore.Mvc.ControllerBase.
